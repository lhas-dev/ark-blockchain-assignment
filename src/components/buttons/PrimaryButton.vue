<template>
  <button class="text-lg bg-red-500 p-3 text-white rounded hover:bg-red-600">
    <slot></slot>
  </button>
</template>

<script>
export default {};
</script>

<style></style>
