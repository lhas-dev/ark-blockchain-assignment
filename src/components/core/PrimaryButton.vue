<template>
  <button class="text-lg"><slot></slot></button>
</template>

<script>
export default {};
</script>

<style></style>
