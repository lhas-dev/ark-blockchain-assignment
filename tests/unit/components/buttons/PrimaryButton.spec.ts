import { shallowMount } from "@vue/test-utils";
import PrimaryButton from "@/components/buttons/PrimaryButton.vue";

describe("PrimaryButton.vue", () => {
  it("renders correctly", () => {
    const wrapper = shallowMount(PrimaryButton, {
      stubs: ["router-link", "router-view"]
    });
    expect(wrapper).toMatchSnapshot();
  });
});
