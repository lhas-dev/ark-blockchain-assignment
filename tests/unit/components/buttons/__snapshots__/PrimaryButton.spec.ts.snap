// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PrimaryButton.vue renders correctly 1`] = `<router-link-stub class="primary-button"></router-link-stub>`;
